variables.tf

```diff
@@ -23,6 +23,12 @@ variable "cluster_config_endpoint_type" {
   }
 }
 
+variable "is_vpc" {
```

Comment:
I wonder would `is_vpc_cluster` more descriptive for consumers?

Reply:
agreed. updated.